release/models/isis/openconfig-isis.yang

```diff
@@ -295,6 +302,19 @@ module openconfig-isis {
     }
   }
 
+  grouping admin-config-deprecated {
+    description
+      "Re-usable grouping for the enabled leaf in contexts where it is deprecated.";
+
+    leaf enabled {
+      type boolean;
+      status deprecated;
+      description
+        "Formerly this leaf was used to enable or disable the functionality
+        within the context specified. This leaf is DEPRECATED.";
+    }
```

Comment:
Was the removal of `default false` on purpose here? Would it make this PR backwards-incompatible since the default statement changed?

@dplore There has already been other backwards-incompatible changes since last release (e.g. https://github.com/openconfig/public/pull/731), so perhaps the comment https://github.com/openconfig/public/pull/794#issuecomment-1420228979 no longer applies? In that case, should we simply remove this grouping altogether as was originally proposed in the PR? Alternatively I think we should add `default false` to match the previous leaf's definition to keep backwards compatibility.

Reply:
Regarding breaking changes: it's true that there are other breaking changes since release 1.0 which have been merged.  I would like to get into the habit of making non-breaking changes when possible.  Although we've had breaking changes merged which requires the next release to be a major release, I would still like to minimize the quantity of breaking changes.    

Unless there is some justification for why this should be a breaking change, I'm biased to make it non-breaking (by submitting a new PR to add `default false`.